bin/automatic_review.py

```diff
@@ -9,18 +9,18 @@
 from egcg_core import rest_communication, clarity
 from egcg_core.constants import ELEMENT_REVIEW_COMMENTS
 from egcg_core.config import cfg
```

Comment:
`egcg_core.config.cfg` is never used here - it goes through `config.load_config` instead. Is this the intended behaviour for now?